libbeat/docs/shared-cloudfoundry.asciidoc

```diff
@@ -0,0 +1,128 @@
+[[running-on-cloudfoundry]]
+=== Run {beatname_uc} on Cloud Foundry
+
+ifeval::["{beatname_lc}"=="filebeat"]
+You can use {beatname_uc} on Cloud Foundry to retrieve and ship logs.
+endif::[]
+ifeval::["{beatname_lc}"=="metricbeat"]
+You can use {beatname_uc} on Cloud Foundry to retrieve and ship metrics.
+endif::[]
+
+ifeval::["{release-state}"=="unreleased"]
+
+However, version {version} of {beatname_uc} has not yet been
+released, no build is currently available for this version.
+
+endif::[]
+
+
+[float]
+==== Cloud Foundry credentials
+
+{beatname_uc} needs credentials created with UAA so it can connect to loggregator to receive the logs. The `uaac`
+command will create the required credentials for connecting to loggregator.
+
+["source","sh",subs="attributes"]
+------------------------------------------------
+uaac client add {beatname_lc} --name {beatname_lc} --secret changeme --authorized_grant_types client_credentials,refresh_token --authorities doppler.firehose,cloud_controller.admin_read_only
+------------------------------------------------
+
+[WARNING]
+=======================================
+*Use a unique secret:* The `uaac` command shown here is an example. Remember to
+replace `changeme` with your secret, and update the +{beatname_lc}.yml+ file to
+use your chosen secret.
+=======================================
+
+
+[float]
```

Comment:
I would remove this float attribute, so the section appears in the table of contents on the right side of the page.